### ISSUE TITLE
Color contrast test picks up `body` element on block themes

### ISSUE THREAD

**Issue by joedolson** [equalizedigital/accessibility-checker#51]
**
![image](https://user-images.githubusercontent.com/1686451/199333300-de64d03a-a7a6-46de-9bef-d28da918ee2b.png)
**

I didn't explore this at length, but tested on Twenty Twenty Two and Twenty Twenty Three, and on both themes the color contrast test picked up the `body` element as a color contrast error. I question how valuable it is to ever return the entire `body` element as an error; but even if that's relevant, the sheer size of the code block is dubious.

The body element is not returned on other default themes that I could see. I'm not sure what would trigger this in your tests, since at least on Twenty Twenty Two, the default color/background is set and is `#fff` on `#000`.

**Comment by SteveJonesDev** [comment]
Can you post the CSS at the bottom of the Affected Code between the curly brackets? I'm guessing it's a false positive due to the plugin having issue interpreting the CSS variables.

**Comment by joedolson** [comment]
```
body{--wp--preset--color--black: #000000; --wp--preset--color--cyan-bluish-gray: #abb8c3; --wp--preset--color--white: #ffffff; --wp--preset--color--pale-pink: #f78da7; --wp--preset--color--vivid-red: #cf2e2e; --wp--preset--color--luminous-vivid-orange: #ff6900; --wp--preset--color--luminous-vivid-amber: #fcb900; --wp--preset--color--light-green-cyan: #7bdcb5; --wp--preset--color--vivid-green-cyan: #00d084; --wp--preset--color--pale-cyan-blue: #8ed1fc; --wp--preset--color--vivid-cyan-blue: #0693e3; --wp--preset--color--vivid-purple: #9b51e0; --wp--preset--color--base: #ffffff; --wp--preset--color--contrast: #000000; --wp--preset--color--primary: #9DFF20; --wp--preset--color--secondary: #345C00; --wp--preset--color--tertiary: #F6F6F6; --wp--preset--gradient--vivid-cyan-blue-to-vivid-purple: linear-gradient(135deg,rgba(6,147,227,1) 0%,rgb(155,81,224) 100%); --wp--preset--gradient--light-green-cyan-to-vivid-green-cyan: linear-gradient(135deg,rgb(122,220,180) 0%,rgb(0,208,130) 100%); --wp--preset--gradient--luminous-vivid-amber-to-luminous-vivid-orange: linear-gradient(135deg,rgba(252,185,0,1) 0%,rgba(255,105,0,1) 100%); --wp--preset--gradient--luminous-vivid-orange-to-vivid-red: linear-gradient(135deg,rgba(255,105,0,1) 0%,rgb(207,46,46) 100%); --wp--preset--gradient--very-light-gray-to-cyan-bluish-gray: linear-gradient(135deg,rgb(238,238,238) 0%,rgb(169,184,195) 100%); --wp--preset--gradient--cool-to-warm-spectrum: linear-gradient(135deg,rgb(74,234,220) 0%,rgb(151,120,209) 20%,rgb(207,42,186) 40%,rgb(238,44,130) 60%,rgb(251,105,98) 80%,rgb(254,248,76) 100%); --wp--preset--gradient--blush-light-purple: linear-gradient(135deg,rgb(255,206,236) 0%,rgb(152,150,240) 100%); --wp--preset--gradient--blush-bordeaux: linear-gradient(135deg,rgb(254,205,165) 0%,rgb(254,45,45) 50%,rgb(107,0,62) 100%); --wp--preset--gradient--luminous-dusk: linear-gradient(135deg,rgb(255,203,112) 0%,rgb(199,81,192) 50%,rgb(65,88,208) 100%); --wp--preset--gradient--pale-ocean: linear-gradient(135deg,rgb(255,245,203) 0%,rgb(182,227,212) 50%,rgb(51,167,181) 100%); --wp--preset--gradient--electric-grass: linear-gradient(135deg,rgb(202,248,128) 0%,rgb(113,206,126) 100%); --wp--preset--gradient--midnight: linear-gradient(135deg,rgb(2,3,129) 0%,rgb(40,116,252) 100%); --wp--preset--duotone--dark-grayscale: url('#wp-duotone-dark-grayscale'); --wp--preset--duotone--grayscale: url('#wp-duotone-grayscale'); --wp--preset--duotone--purple-yellow: url('#wp-duotone-purple-yellow'); --wp--preset--duotone--blue-red: url('#wp-duotone-blue-red'); --wp--preset--duotone--midnight: url('#wp-duotone-midnight'); --wp--preset--duotone--magenta-yellow: url('#wp-duotone-magenta-yellow'); --wp--preset--duotone--purple-green: url('#wp-duotone-purple-green'); --wp--preset--duotone--blue-orange: url('#wp-duotone-blue-orange'); --wp--preset--font-size--small: clamp(0.875rem, 0.875rem + ((1vw - 0.48rem) * 0.24), 1rem); --wp--preset--font-size--medium: clamp(1rem, 1rem + ((1vw - 0.48rem) * 0.24), 1.125rem); --wp--preset--font-size--large: clamp(1.75rem, 1.75rem + ((1vw - 0.48rem) * 0.24), 1.875rem); --wp--preset--font-size--x-large: 2.25rem; --wp--preset--font-size--xx-large: clamp(4rem, 4rem + ((1vw - 0.48rem) * 11.538), 10rem); --wp--preset--font-family--dm-sans: 'DM Sans'; --wp--preset--font-family--ibm-plex-mono: 'IBM Plex Mono'; --wp--preset--font-family--inter: Inter; --wp--preset--font-family--system-font: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif; --wp--preset--font-family--source-serif-pro: 'Source Serif Pro'; --wp--preset--spacing--30: clamp(1.5rem, 5vw, 2rem); --wp--preset--spacing--40: clamp(1.8rem, 1.8rem + ((1vw - 0.48rem) * 2.885), 3rem); --wp--preset--spacing--50: clamp(2.5rem, 8vw, 4.5rem); --wp--preset--spacing--60: clamp(3.75rem, 10vw, 7rem); --wp--preset--spacing--70: clamp(5rem, 5.25rem + ((1vw - 0.48rem) * 9.096), 8rem); --wp--preset--spacing--80: clamp(7rem, 14vw, 11rem); margin: 0; --wp--style--global--content-size: 650px; --wp--style--global--wide-size: 1200px; --wp--style--block-gap: 1.5rem; background-color: --wp--preset--color--base; color: var(--wp--preset--color--contrast); font-family: var(--wp--preset--font-family--system-font); font-size: --wp--preset--font-size--medium; line-height: 1.6; --wp--style--root--padding-top: var(--wp--preset--spacing--40); --wp--style--root--padding-right: var(--wp--preset--spacing--30); --wp--style--root--padding-bottom: var(--wp--preset--spacing--40); --wp--style--root--padding-left: var(--wp--preset--spacing--30); }
```

**Comment by SteveJonesDev** [comment]
As I suspected it's most likely an issue interpreting the CSS variables. You do bring up a good point on returning the whole body element. Tagging this as an enhancement and will look into it further.

Thanks @joedolson!

**Comment by SteveJonesDev** [comment]
Fixed with v1.7.0.
